andela/travel_tool_front
# #161755477 Fix checkbox misalignment

#### What does this PR do?
Updates the checkbox alignment on the request modal

#### Description of Task to be completed?
- set checkbox description alignment to the right of the checkbox

#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_front.git`
- Switch to the project directory - `cd travel_tool_front`
- Checkout - `git checkout bg-fix-checkbox-misalignment-161755477`
- Start the app - `make start`
- Start the backend of this project - `git clone https://github.com/andela/travel_tool_back.git && make start`
- Navigate to travela in your browser and log in
- Navigate to the new request modal
- You should see the checkbox description properly aligned to the right

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#161755477](https://www.pivotaltracker.com/story/show/161755477)

#### Screenshots (if appropriate)
<img width="1168" alt="screen shot 2018-11-08 at 11 58 03 am" src="https://user-images.githubusercontent.com/13545380/48194537-99f8b380-e34d-11e8-9da8-27bafee5765e.png">


#### Questions:
